src/lib/holocene/combobox/combobox.svelte

```diff
@@ -292,17 +295,24 @@
       bind:this={inputElement}
       {...$$restProps}
     />
-    <button
+    <Button
       aria-label={toggleLabel}
       class="combobox-button"
```

Comment:
Looks like this `combobox-button` class isn't being applied 🤔 The `border-radius` appears to not be `rounded-full`.

<img width="313" alt="Screenshot 2024-05-21 at 2 35 29 PM" src="https://github.com/temporalio/ui/assets/15069288/c5d79f0a-c2b5-4538-a519-65505d35d8f1">

Reply:
I just removed the class altogether since it's not being applied. I think we should use the ghost button defaults since it's a button.